journal.py

```diff
@@ -54,6 +56,22 @@ def all(cls, session=None):
             session = DBSession
         return session.query(cls).order_by(cls.created.desc()).all()
 
+    @classmethod
+    def get_entry(cls, entry_id, session=None):
+        if session is None:
+            session = DBSession
+        entry = session.query(cls).filter(Entry.id == entry_id).first()
+        return entry.title, entry.text, entry.created.strftime('%b. %d, %Y')
```

Comment:
Think a bit about how it will look when you call this method:

```python
entry = Entry.get_entry(42)
```

It feels to me like the `entry` part of the method name is pretty well covered by the fact that you call the method on the Entry class.  

This is a nit, but I like method names that make for clear, concise and easy-to-read code.